### PR TITLE
Feature/fwf 3426 navigation changes

### PR DESCRIPTION
**Pull request by Ajay-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3426
Issue Type: BUG/ FEATURE

# Changes
1.New navigation changes
2.permission matrix changes added

### CODE REVIEW

File: forms-flow-nav/src/sidenav/hamburger.scss
Comment (arun-s-aot) [inline forms-flow-nav/src/sidenav/hamburger.scss:1]: Avoid writing same css selector multiple times. (Ex : .custom-navbar )
Avoid writing duplicate media query tags with same breakpoint

File: forms-flow-nav/src/sidenav/Sidebar.scss
Comment (arun-s-aot) [inline forms-flow-nav/src/sidenav/Sidebar.scss:1]: Please move the root variables to theme.scss file which can be used commonly in other pages

**Review by arun-s-aot — CHANGES_REQUESTED** [review]
Please make changes as per the comments

File: forms-flow-nav/src/sidenav/Sidebar.scss
Comment (Josephalexantony-aot) [reply forms-flow-nav/src/sidenav/Sidebar.scss:1]: Use the colors from the color system instead of using direct hexadecimal or HSL

Comment (Ajay-aot) [reply forms-flow-nav/src/sidenav/Sidebar.scss:1]: done

File: forms-flow-nav/src/sidenav/hamburger.scss
Comment (Ajay-aot) [reply forms-flow-nav/src/sidenav/hamburger.scss:1]: done

**Review by arun-s-aot — APPROVED** [review]

**Review by Josephalexantony-aot — APPROVED** [review]

**Pull request merged**